client/e2e/pages/EventsPage.ts

```diff
@@ -193,34 +227,58 @@ export class EventsPage {
     await this.editTicketQuantity.fill(showing.showingQuantity);
     await this.newEventSave.click();
     await this.eventContinue.click();
+   // await this.page.reload();
   }
 
   /**
    * We need to pass in things to the filter like:
    * "Wed, Oct 11 2023"
    */
-  async searchDeleteShowing(time: string) {
-    await this.showingCard
-      .filter({hasText: time})
-      .getByRole('button', {name: 'Edit'})
-      .click();
+  async searchDeleteShowing(Time: string) {
+   const showingCardLocator =  this.showingCard
+      .filter({hasText: Time})
+      .getByRole('button', {name: 'Edit'}); 
+   const disabled = await showingCardLocator.isDisabled();
+    //console out the value of the disabled//
+    console.log("disabled is: " + disabled); 
+    if (disabled)
+    {
+      await this.page.reload();
+    }
+    await showingCardLocator.click();
     await this.deleteShowingButton.click();
     await this.eventContinue.click();
   }
 
-  //**The weird thing is that Im not sure wheres the 'Playbill' comes from.
-  async checkNewEventOnHomePage() {
+  /**
+   * 1.Whenever we created a new event and go back to the homepage 
+   * since the homepage needs time to update, but the tests cant wait for too long
+   * We need to click the "cart" button in order to refresh the homepage then we can 
+   * see our newly created event on homepage now 
+   * 2. We need to wait for few seconds each time we click the slide
+   * since if we click it too fast, the image on homepage could only move 
+   * very little distance
+   */
+  async checkNewEventOnHomePage(event_name:string, suffix:string) {
     await this.homePage.click();
+    await this.page.waitForTimeout(100);
+    await this.cartButton.click();
+    await this.goToHomepageFromCart.click();
+    await this.page.waitForTimeout(400);
     await this.homePageRightSlide.click();
+    await this.page.waitForTimeout(400);
     await this.homePageRightSlide.click();
+    await this.page.waitForTimeout(400);
     await this.homePageRightSlide.click();
+    await this.page.waitForTimeout(400);
     await this.homePageRightSlide.click();
+    await this.page.waitForTimeout(400);
     await this.homePageRightSlide.click();

```

Comment:
This may only work for one specific event. Perhaps a loop can be made? Scroll until you see? This one might be tough. Concern is number of homePageRightSlide is fixed and inflexible here. Is the sliding needed for playwright to detect  the event card?

Reply:
I think it doesn't matter for now, even though there's no event at all before we created a new event, you can always scroll 5 times to see the newly-created event.  The only exception is that there are much more events than our website currently has on the homepage, probably scroll 5 times is not enough to see the event we want. I will try to achieve it in another way, but for now, I dont think its not robust, this function must be only work for one specific event, the specific event is that the one we just created!

Reply:
does it work without slidiing? what if instead of checking if it is visible, just check if it exists. (line 278)